test/test_service.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/python
+# coding=utf-8
+
+import json
+
+
+def test_download_mnist(mnist_class):
```

Comment:
This is probably already tested within Keras/Tensorflow

Reply:
I have put it in to check that the MNIST dataset isn't updated by tensorflow at some point in the future, meaning the syntax of this quickstart will no work any more. 

I think I will leave it in for that reason